tests/performance/scripts/runperf.sh

```diff
@@ -37,8 +37,8 @@ cat >config.json<<END
         "node_num": 10,
         "imagerepo": "pavan187",
         "k8smaster2": "http://$MASTER_IP_2:12458",
-        "cloudimageurl": "pavan187/cloudcore:v1.0",
-        "edgeimageurl": "pavan187/edgecore:latest",
+        "cloudimageurl": "pavan187/cloudcore:v2.1",
```

Comment:
can we start pushing images to kubeedge dockerhub and maybe rename as kubeedge/cloudcore:<version>?

Reply:
Okay. Discussed with @kevin-wangzefeng .
He helped to create some repostiory under kubeedge.
We will use `kubeedge/edgecore-test` and `kubeedge/edgecontroller-test` here.